packages/web/pages/settings/account.tsx

```diff
@@ -441,19 +442,23 @@ export default function Account(): JSX.Element {
             <StyledLabel>Beta features</StyledLabel>
             {!isValidating && (
               <>
-                {viewerData?.me?.features.map((feature) => {
+                {viewerData?.me?.featureList.map((feature) => {
```

Comment:
Does featurelist only include a feature if its granted?

Reply:
No, it includes all the features